les/fetcher.go

```diff
@@ -50,7 +50,7 @@ type lightFetcher struct {
 	lastUpdateStats *updateStatsEntry
 	syncing         bool
 	syncDone        chan *peer
-
+  SyncedCh      chan bool
```

Comment:
Example of formatting error

Reply:
Fixed individual issue and also utilized gofmt